### DISCUSSION TITLE
How to remove icons from grid view

### DISCUSSION THREAD

**Discussion by Hbitvof** [pimcore/pimcore#12180] [Q&A / Support]
Hi,

I want to remove some icons from the default grid view:

![image](https://user-images.githubusercontent.com/41229269/169643162-42349fff-ddc4-4287-bb8c-fc900eb9ac8a.png)

I can't find any documentation where to configure this. Anyone know how to do this?

**Comment by Hbitvof** [comment]
Anyone?

**Comment by kingjia90** [comment]
It seems something that must be done by customizing Extjs, so you may need to search in their documentation there, by `overriding` the js code that creates and defines the `iconCls` of the button, but i don't know specifically how do it, hope it helps in finding a solution.
